query-node/manifest.yml

```diff
@@ -2,37 +2,41 @@ version: '0.1'
 description: Joystream query-node manifest file for sumer
 repository: https://github.com/Joystream/joystream
 dataSource:
-  kind: substrate 
+  kind: substrate
   chain: joystream
   indexerVersion: '0.1.6'
```

Comment:
`data.indexerVersion` and `mappings.hydraCommonVersion` should be removed and replaced with a single top-level `hydraVersion: "2"`